src/oic/oic/__init__.py

```diff
@@ -759,6 +759,9 @@ def user_info_request(self, method="GET", state="", scope="", **kwargs):
             else:
                 # use_authorization_header, token_in_message_body
                 if "use_authorization_header" in _behav:
+                    if _ttype.lower() == _ttype:
+                        # fix 'bearer' to 'Bearer' (standard requirement)
+                        _ttype = _ttype.capitalize()
                     token_header = "{type} {token}".format(type=_ttype,
```

Comment:
Yes, in this case it works of course.
My initial idea was to add a warning here and hunt out all source places where this problem is coming from, but I haven't had time and have forgotten about it.
Not sure we aren't having any other problems because lowercase word is being sent from somewhere else.